Sources/LogtoClient/LogtoAuthSession/LogtoWebViewAuthViewController+WKScriptMessageHandler.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  LogtoWebViewAuthViewController+WKScriptMessageHandler.swift
+//  
+//
+//  Created by Gao Sun on 2022/4/19.
+//
+
+import Foundation
+import WebKit
+
+extension LogtoWebViewAuthViewController: WKScriptMessageHandler {
+    struct SocialPostBody: Codable {
+        static func safeParseJson(json: Any) -> SocialPostBody? {
+            guard let data = try? JSONSerialization.data(withJSONObject: json) else {
+                return nil
+            }
+            return try? JSONDecoder().decode(self, from: data)
```

Comment:
Noob question: where does this `self` come from?

Reply:
yes